### PR TITLE
feat: Add Linea Sepolia chain

### PR DESCRIPTION
**Pull request by VGau**
## PR-Codex overview
This PR adds support for Linea Sepolia and updates Linea Goerli in the smart timers project.

### Detailed summary
- Added `lineaSepolia` chain definition
- Updated `lineaGoerli` chain definition with new details

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/lineaGoerli.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { defineChain } from '../../utils/chain/defineChain.js'
 
-export const lineaTestnet = /*#__PURE__*/ defineChain({
+export const lineaGoerli = /*#__PURE__*/ defineChain({
```
Comment (jxom) [inline src/chains/definitions/lineaGoerli.ts:3]: This is a breaking change. I suggest the following:

1. Add a new file `lineaGoerli.ts` with these contents.
2. Mark the `lineaTestnet` export as deprecated in `src/chains/index.ts` with a `/** @deprecated Use `lineaGoerli` */` comment

This means we can remove this export and file in Viem v3 (next major version).

**Review by jxom — CHANGES_REQUESTED** [review]

File: src/chains/definitions/lineaGoerli.ts (hunk 1 above)
Comment (VGau) [reply src/chains/definitions/lineaGoerli.ts:3]: Okay I will do that. Thanks :)

Comment (jxom) [reply src/chains/definitions/lineaGoerli.ts:3]: Legend.

Comment (VGau) [reply src/chains/definitions/lineaGoerli.ts:3]: Done :)

**Pull request merged**